### PR TITLE
[REF] point_of_sale,*: more consistent test files structure

### PR DESCRIPTION
**Pull request by SalukiMakingCode**
*: all pos-related modules

1. Use snake_case on file names similar to how we name other files.
2. Tour files are suffixed with `_tour`.
3. Helper files are moved to `/tours/utils` and are suffixed with `_util`.

tasks id : 3794884

Linked to: https://github.com/odoo/enterprise/pull/58396

### CODE REVIEW

File: addons/l10n_be_pos_sale/static/tests/tours/l10n_be_pos_sale_tour.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /** @odoo-module */
-import * as PaymentScreen from "@point_of_sale/../tests/tours/helpers/PaymentScreenTourMethods";
-import * as ProductScreenPos from "@point_of_sale/../tests/tours/helpers/ProductScreenTourMethods";
+import * as PaymentScreen from "@point_of_sale/../tests/helpers/PaymentScreenTourMethods";
+import * as ProductScreenPos from "@point_of_sale/../tests/helpers/ProductScreenTourMethods";
```
[the remark this answers is not in the document]
Comment (vlst-odoo) [reply addons/l10n_be_pos_sale/static/tests/tours/l10n_be_pos_sale_tour.js:3]: Thank you for this response, but i could have asked chatgpt directly for it. I wanted to know what's the advantage of moving helpers that are used only in the tours in the main `tests` folders and I still don't have an answer for this. The `test` folder contains the subfolders `unit` and `tours`. If i have `tour` specific helpers, why not leave them in the `tests/tours` folder ?

[the remark this answers is not in the document]
Comment (SalukiMakingCode) [reply addons/l10n_be_pos_sale/static/tests/tours/l10n_be_pos_sale_tour.js:3]: Firstly, avoid splitting the helpers. Keep all helpers within the 'helpers' folder and all tours within the 'tours' folder for greater readability. Secondly, maintain the same organization across all modules. Thirdly, I've briefly looked into how other teams handle this, and they all seem to follow this approach. And fourthly, when you go to the tests folders, you can understand in one second that there are 'tours' and 'helpers' to make the test work.

File: addons/point_of_sale/static/tests/tours/helpers/generic_components/order_widget_helpers.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /** @odoo-module */
 
-import { negate } from "@point_of_sale/../tests/tours/helpers/utils";
+import { negate } from "@point_of_sale/../tests/tours/helpers/utils_helpers";
```
Comment (vlst-odoo) [inline addons/point_of_sale/static/tests/tours/helpers/generic_components/order_widget_helpers.js:3]: no need for this change. The name `utils` is clear enough

Comment (SalukiMakingCode) [reply addons/point_of_sale/static/tests/tours/helpers/generic_components/order_widget_helpers.js:3]: I agree with you, this is clear. But it's not about beeing clear, but about follow the guideline. What do you think ?

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**